script.js

```diff
@@ -1,4 +1,654 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
+var userScore = 0;
+var noOfGame = 0;
+var userDraw = 0;
+var nickName = " ";
+var gameMode = "normal";
+var playMode = "normal";
+var koreanMode = 0;
+var koreanLoseMode = 0;
+var koreanWinMode = 0;
+var comAScore = 0;
+var comBScore = 0;
+// random number generate
+var randomNumber = function () {
+  var randomDice = Math.random() * 3;
+  var randomIntegal = Math.ceil(randomDice);
+  var actualResult = randomIntegal;
+  if (actualResult == 1) {
+    return "scissors";
+  }
+  if (actualResult == 2) {
+    return "paper";
+  }
+  if (actualResult == 3) {
+    return "stone";
+  }
+};
```

Comment:
Nice job decomposing the logic out :) !
One note. Our functions should be named **starting with a verb** and also **describe what it does**.

For example this function can be name `getScissorsPaperOrStone `or `getSPS`